chain/client/src/client_actor.rs

```diff
@@ -164,6 +170,14 @@ impl Handler<NetworkClientMessages> for ClientActor {
             #[cfg(feature = "adversarial")]
             NetworkClientMessages::Adversarial(adversarial_msg) => {
                 return match adversarial_msg {
+                    AdvDisableDoomslug => {
+                        info!(target: "adversary", "Turning Doomslug off");
+                        self.adv_disable_doomslug = true;
+                        self.client.doomslug.adv_disable();
+                        self.client.chain.doomslug_threshold_mode =
```

Comment:
instead add `adv_disable` to chain, that changes its local, and calls to doomslug? Seems nice because
a) makes only one call here, so it's better isolation
b) removes the necessity for the pub/non-pub mod above

Reply:
I'll add two methods: `doomslug.adv_disable()` and `chain. adv_disable_doomslug()`